app/models/nonvolatile_memory express_address.rb

```diff
@@ -0,0 +1,7 @@
+class NonvolatileMemoryExpressAddress < SanAddress
```

Comment:
`NvmeAddress` ?

Reply:
> `NvmeAddress` ?

yes, I wasn't sure how to write the short version because of ruby naming conventions. I dont think `NVMeAddress` is preferred because the file name will be `n_v_me_address`, but `NvmeAddress` is possible. do you suggest to use `NvmeAddress` or the full name like I did `NonvolatileMemoryExpressAddress` ?

Reply:
My two cents is `NvmeAddress`

Reply:
I'm with you, changed to NvmeAddress.